erizo/src/erizo/media/ExternalOutput.cpp

```diff
@@ -315,7 +363,11 @@ int ExternalOutput::deliverAudioData_(char* buf, int len) {
 int ExternalOutput::deliverVideoData_(char* buf, int len) {
     if (videoSourceSsrc_ == 0){
       RtpHeader* h = reinterpret_cast<RtpHeader*>(buf);
-      videoSourceSsrc_ = h->getSSRC();
+      if (h->getPayloadType() == RED_90000_PT) {
```

Comment:
If you do turn it off, you'll also need a check to see if the payload type is VP8/VP9 or some other video codec